src/main/java/blackjack/domain/result/DealerResult.java

```diff
@@ -0,0 +1,30 @@
+package blackjack.domain.result;
+
+import java.util.Collections;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+
+public class DealerResult {
+    private final Map<Result, Integer> result;
+
+    public DealerResult(final List<PlayerResult> playerResults) {
+        this.result = new EnumMap<>(Result.class);
+        calculateResultCount(playerResults);
+    }
+
+    private void calculateResultCount(final List<PlayerResult> playerResults) {
+        for (final PlayerResult playerResult : playerResults) {
+            addCount(Result.reverse(playerResult.getResult()));
```

Comment:
```suggestion
            addCount(playerResult.reversed());
            
            class PlayerResult {
              ... 
              public Result reversed() {
                 return this.result.reverse() 
              }
              ...
            }
```

`Result.reverse()` static 메서드를 인스턴스 메서드로 만들고, DealerResult 객체가 PlayerResult 객체에게 메시지를 보내서 반전된 결과를 가져오도록 하면 어떨까요?

결과를 반전시키기 위해서 Result에 static 메서드가 있음을 DealerResult가 알아야 하는 것과, 알고 있는 playerResult에게 메시지를 보내서 객체간 협력하게 만드는 것은 의미있는 차이가 있다고 봅니다 :)